vulnfeeds/cmd/alpine/verify.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "regexp"
+
+func valid_version(ver string) bool {
+	// Following the version requirement specified here: https://github.com/alpinelinux/abuild/blob/master/APKBUILD.5.scd
+	checker, err := regexp.Compile(
```

Comment:
nit: Just use regexp.MustCompile so we don't have to add the error handling ourselves.